e2e/utils.go

```diff
@@ -1276,3 +1273,27 @@ func retryKubectlFile(namespace string, action kubectlAction, filename string, t
 		return true, nil
 	})
 }
+
+// createSecret creates secret given namespace and secret yaml file path.
+func createSecret(c kubernetes.Interface, secretPath, ns string) {
+	secret, err := getSecret(secretPath)
+	if err != nil {
+		e2elog.Failf("failed to load secret: %v", err)
+	}
+	_, err = c.CoreV1().Secrets(ns).Create(context.TODO(), &secret, metav1.CreateOptions{})
+	if err != nil {
+		e2elog.Failf("failed to create Secret %q: %v", err, secret.Name)
+	}
+}
+
+// deleteSecret deletes secret given namespace and secret yaml file path.
+func deleteSecret(c kubernetes.Interface, secretPath, ns string) {
+	secret, err := getSecret(secretPath)
+	if err != nil {
+		e2elog.Failf("failed to load secret: %v", err)
+	}
+	err = c.CoreV1().Secrets(ns).Delete(context.TODO(), secret.Name, metav1.DeleteOptions{})
+	if err != nil {
+		e2elog.Failf("failed to create Secret %q: %v", err, secret.Name)
+	}
+}
```

Comment:
create a new file secret.go and move this helper to a secret file and return an error from the helper let the caller handle it.

Reply:
if these helpers are needed at all...